docker/nginx/conf.d/include/track-upload

```diff
@@ -5,14 +5,13 @@ log_by_lua_block {
 
     local skylink = ngx.header["Skynet-Skylink"]
     if skylink and ngx.status >= ngx.HTTP_OK and ngx.status < ngx.HTTP_SPECIAL_RESPONSE then
-        local http = require("socket.http")
-        local ok, statusCode, headers, statusText = http.request {
-            url = "http://accounts:3000/track/upload/" .. skylink,
+        local httpc = require("resty.http").new()
+        local res, err = httpc:request_uri("http://10.10.10.70:3000/track/upload/" .. skylink, {
             method = "POST",
-            headers = ngx.req.get_headers()
-        }
-        if statusCode ~= ngx.HTTP_NO_CONTENT and statusCode ~= ngx.HTTP_UNAUTHORIZED then
-            ngx.log(ngx.ERR, "accounts endpoint /track/upload/" .. skylink .. " failed with error " .. statusCode)
+            headers = { ["Cookie"] = ngx.header["Cookie"] },
```

Comment:
This syntax seems so off... I'm not saying it's wrong, I'm just saying it looks very weird to me, so please take a look again. :)

Reply:
yeah I know :) but this syntax is valid lua

![Screenshot 2021-08-24 at 13 25 59](https://user-images.githubusercontent.com/3755029/130608524-814a42fb-d2d9-4567-990a-82e1b9b50361.png)

Reply:
Oh boy... :D